### ISSUE TITLE
Not a Siamese network?

### ISSUE THREAD

**Issue by yiboin** [D-X-Y/caffe-reid#3]
Hi,
   I cannot find the verification loss  in the train.proto .Is this implementation(network) same with the paper?

**Comment by D-X-Y** [comment]
See https://github.com/D-X-Y/caffe-reid/blob/master/models/market1501/vgg_reduce/train.proto
The 2-way softmax loss "loss_dif" is the verification loss.
